vault/token_store.go

```diff
@@ -2024,6 +2115,15 @@ func (ts *TokenStore) tokenStoreRoleCreateUpdate(
 }
 
 const (
+	tokenCleanupHelp = `
+This endpoint cleans up the storage and lease entries.
+`
+	tokenCleanupDesc = `
+This is provided to be able to cleanup accessor storage entries that were
```

Comment:
I'd leave it less specific as we may add things later. Something like "This endpoint performs cleanup tasks that can be run to clean up token and lease entries after certain error conditions. Usually running this is not necessary, and is only required if upgrade notes or support personnel suggest it."